docs/api/cors.rst

```diff
@@ -16,12 +16,17 @@ By default, Falcon's built-in CORS support is disabled, so that any cross-origin
 requests will be blocked by the browser. Passing ``cors_enable=True`` will
 cause the framework to include the necessary response headers to allow access
 from any origin to any route in the app. Individual responders may override this
-behavior by setting the Access-Control-Allow-Origin header explicitly.
+behavior by setting the ``Access-Control-Allow-Origin`` header explicitly.
 
 Whether or not you implement a CORS policy, we recommend also putting a robust
 AuthN/Z layer in place to authorize individual clients, as needed, to protect
 sensitive resources.
 
+Directly passing the :class:`falcon.CORSMiddleware` middleware to the application
+allows customization of the CORS policy applied. The middleware allows customizing
+the allowed origins, if credentials should be allowed and if additional headers
```

Comment:
The sentence structure got a bit broken here (unless I'm reading it totally wrong):
> and if additional headers ~~that~~ can be exposed